packages/squiggle-lang/src/ast/serialize.ts

```diff
@@ -220,7 +222,7 @@ export function deserializeAstNode(
         ...node,
         imports: node.imports.map((item) => [
           visit.ast(item[0]) as KindNode<"String">,
-          visit.ast(item[0]) as KindNode<"Identifier">,
+          visit.ast(item[1]) as KindNode<"Identifier">,
```

Comment:
Unrelated fix, but obviously a typo. This part is rewritten in the upcoming semantic analysis PR, anyway.